bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/types/ArithmeticGroupFunction.java

```diff
@@ -573,6 +580,20 @@ public State getStateAs(Set<Item> items, Class<? extends State> stateClass) {
         public State[] getParameters() {
             return new State[] { upperState, lowerState, upperLimit, lowerLimit };
         }
+        
+		private String getFunctype(Item item) {
+			String memberFuncType = "Undef";

```

Comment:
If we change this method to return boolean or Class, we wouldn’t need this magic constant. The default return values would be `false` and `DecimalType.class` respectively.